ci/test_wheel.sh

```diff
@@ -3,14 +3,37 @@
 
 set -euo pipefail
 
+CUDA_VER_MAJOR_MINOR=${CUDA_VER%.*}
+
 rapids-logger "Install testing dependencies"
 # TODO: Replace with rapids-dependency-file-generator
 python -m pip install \
     psutil \
     cffi \
-    cuda-python \
+    "cuda-python==${CUDA_VER_MAJOR_MINOR}.*" \
```

Comment:
I think this may have to be just the major version or parsed such that we get `12.*`
```
ERROR: No matching distribution found for cuda-python==12.2.*
```

Would something like this work?

```suggestion
    "cuda-python==${CUDA_VER%%.*}.*" \
```

Reply:
I thought the problem was that there aren't old cuda-python versions for these newer Python versions, so I was looking at redistributing the test matrices so that the version constraints can be satisfied.

Reply:
Please use `12.*` or equivalent here. It is recommended to use the newest cuda-python even with older CTK versions. There is only a major (not minor) `cuda-version` constraint for `cuda-python`.